server/src/user-org/user-org.service.ts

```diff
@@ -15,6 +15,7 @@ export class UserOrganizationsService {
   ) {}
 
   async create(createUserOrganizationDto: CreateUserOrganizationDto): Promise<UserOrganization> {
+    console.log(createUserOrganizationDto, 'dot from userorg');
```

Comment:
remove log unless you are intentionally keeping it in

Reply:
I removed this